internal/scaffold/ansible/molecule_test_local_prepare.go

```diff
@@ -67,4 +68,4 @@ const moleculeTestLocalPrepareAnsibleTmpl = `---
       - role.yaml
       - role_binding.yaml
       - service_account.yaml
-`
+` //nolint:golint
```

Comment:
When we use `//nolint:golint` we should pass what lint it should ignore. For example: `//nolint:golint,unused` . otherwise, all checks will be ignored.